pkg/migrations/op_alter_column.go

```diff
@@ -189,9 +189,9 @@ func (o *OpAlterColumn) Validate(ctx context.Context, s *schema.Schema) error {
 		return ColumnDoesNotExistError{Table: o.Table, Name: o.Column}
 	}
 
-	// Ensure that the column has a primary key defined on exactly one column.
-	err := checkBackfill(table)
-	if err != nil {
+	// If the operation requires backfills (ie it isn't a rename-only operation),
+	// ensure that the column meets the requirements for backfilling.
+	if err := checkBackfill(table); !o.isRenameOnly() && err != nil {
```

Comment:
should we avoid calling checkBackfill if we don't plan to backfill?

Reply:
Changed this so that we avoid the check entirely for rename only column alters.